OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/DialogErrorNotification.java

```diff
@@ -45,14 +46,16 @@ public void showError(@StringRes final int errorResId,
                           @Nullable final Icon icon,
                           @StringRes final int actionTextResId,
                           @Nullable final View.OnClickListener actionListener) {
-        AlertDialogFragment.newInstance(0, errorResId,
-                actionListener == null ? null :
-                        new DialogInterface.OnClickListener() {
-                            @Override
-                            public void onClick(DialogInterface dialog, int which) {
-                                actionListener.onClick(((AlertDialog) dialog).getButton(AlertDialog.BUTTON_POSITIVE));
+        if (baseFragment.isResumed()) {
+            AlertDialogFragment.newInstance(0, errorResId,
+                    actionListener == null ? null :
+                            new DialogInterface.OnClickListener() {
+                                @Override
+                                public void onClick(DialogInterface dialog, int which) {
+                                    actionListener.onClick(((AlertDialog) dialog).getButton(AlertDialog.BUTTON_POSITIVE));
+                                }
                             }
-                        }
-        ).show(fragmentManager, null);
+            ).show(baseFragment.getFragmentManager(), null);
```

Comment:
Any reason why we are using `getFragmentManager()` whereas previously we always passed `getChildFragmentManager()` ?